### PR TITLE
Rename default branch to main

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Rename the default branch to main in the Github workflows

### How to review
Github actions pass

### CODE REVIEW

**Comment by rmccar** [comment]
[eq-questionnaire-launcher rename default branch](https://trello.com/c/UCIj07E8/5596-eq-questionnaire-launcher-rename-default-branch)

**Review by Yuyuutsu — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Main
```
Comment (creativeappdevamit) [inline .github/workflows/main.yml:1]: why is this all showing new entries? Is it something to do with Git?

**Review by creativeappdevamit — COMMENTED** [review]
Tested looks good to me .. but I have one question of why its being shown new entries.

File: .github/workflows/main.yml (hunk 1 above)
Comment (rmccar) [reply .github/workflows/main.yml:1]: Sorted

Comment (creativeappdevamit) [reply .github/workflows/main.yml:1]: 👍

**Review by creativeappdevamit — APPROVED** [review]

**Pull request merged**